pkg/policy/ebpf.go

```diff
@@ -318,6 +318,15 @@ func (ps *policies) createNewEventsMapVersion(
 		// encoded event's field types
 		var fieldTypes uint64
 		fields := eventsFields[id]
+
+		/*
+			each event may have at most 8 argument data types stored.
+			in the map, the arguments are stored in a 64bit sized bitmap where each 8 bits
+			represent some argument type.
+			For example consider an event with two int arguments (argType = 1) and one string
+			(argType = 10):
+			0 0 0 0 0 a 1 1 - would be the encoded argument types bitmap
+		*/
```

Comment:
Just a rewrite suggestion, be my guest to ignore it.

```suggestion
		/*
			Each event can have up to 8 argument data types stored.
			These argument types are encoded in a 64-bit bitmap where:
			- Each byte (8 bits) represents a single argument type.
			- Therefore, a maximum of 8 argument types can be represented.

			For example, consider an event with:
			- Two integer arguments (argType = 1)
			- One string argument (argType = 10)

			The bitmap would be encoded as (in hex representation):
			00 00 00 00 00 0A 01 01 (0x00000000000A0101)

			Breaking down the bitmap from right to left (least significant byte first):

			| Byte Position | Argument Type          |
			|---------------|------------------------|
			| 0             | 0x01 (int argument)    |
			| 1             | 0x01 (int argument)    |
			| 2             | 0x0A (string argument) |
			| 3 - 7         | 0x00 (padding, unused) |

		*/
```